api/overdrive.py

```diff
@@ -55,6 +55,7 @@ class OverdriveAPI(BaseOverdriveAPI, BaseCirculationAPI):
         { "key": BaseOverdriveAPI.WEBSITE_ID, "label": _("Website ID") },
         { "key": ExternalIntegration.USERNAME, "label": _("Client Key") },
         { "key": ExternalIntegration.PASSWORD, "label": _("Client Secret") },
+        { "key": BaseOverdriveAPI.ILS_NAME, "label": _("ILS Name") },
```

Comment:
This is fine, but you could also put `"optional": True` or `"default": "default"` in here, up to you.